client/app/(todos)/components/Alert.tsx

```diff
@@ -17,7 +17,7 @@ const Alert = ({
   <div className='flex flex-row gap-3 rounded-sm border-[1px] border-red-400 bg-red-100 px-5 py-6 md:gap-6 md:px-8'>
     <FiAlertTriangle className='h-6 w-6 text-red-800' />
     <div className='flex flex-col items-start gap-1 md:gap-3'>
-      <p className='md:text-md text-sm font-semibold'>{mainMessage}</p>
+      <p className='text-sm font-semibold md:text-base'>{mainMessage}</p>
```

Comment:
The 16px font size did not work so I fixed here too
<img width="826" alt="Screenshot 2024-04-12 at 8 51 57 PM" src="https://github.com/HAK2024/fairy-share/assets/66394413/71564038-3459-4189-a45b-29f905ecf8e2">